docs/src/atopics.md

```diff
@@ -2,7 +2,7 @@
 
 ```@example 1
 import JSServe # hide
-JSServe.Page(exportable=true, offline=true) # hide
+JSServe.Page() # hide
```

Comment:
I think this should be
```suggestion
JSServe.Page(;exportable=true, offline=true, current_page_dir= abspath(pwd())*"/atopics/") # hide
```

Analogue patch for the other example.

Reply:
I think the `current_page_dir` is not necessary